market/utils/axonivy-express/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+	"id": "axonivy-express",
+	"name": "Axon Ivy Express",
+	"description": "No-code tool for business users to create and share basic process solutions.",
```

Comment:
Hi @andreasbalsiger, @ivy-sgi 
Could you please help me review the description and tags, ...?
Feel free to suggest a better sentence here. This one will be shown in the summary of the market product.

Reply:
suggestion for description text: 
Simple process app creation for business users - of course, no-code-based.

Reply:
Thanks for your suggestion. @andreasbalsiger 
Yes, I will update it.